src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,220 @@
+---
+title: SplitChunksPlugin
+contributors:
+  - sokra
+  - jeremenichelli
+related:
+  - title: "webpack 4: Code Splitting, chunk graph and the splitChunks optimization"
+    url: https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
+---
+
+Originally in webpack, chunks (and modules import inside them) were connected by a parent-child relationship in the internal webpack graph.
+
+This connection didn't allow to further optimizations and reluted into more coded downloaded.
```

Comment:
s/reluted/resulted

Reply:
s/more coded downloaded/more downloaded code

Reply:
s/allow to/allow for/